doc/ref_model/chapters/chapter07.rst

```diff
@@ -411,35 +432,77 @@ To secure software code, the following methods must be applied:
-
-`SPDX <https://spdx.dev/>`__ is an open-source machine-readable format developed under the umbrella of the Linux Foundation. The `SPDX specification 2.2 <https://spdx.dev/specifications/>`__ has been published as the standard ISO/IEC 5962:2021. It provides a language for communicating the data, licenses, copyrights, and security information associated with software components. With the SPDX specification 2.2, multiple file formats are available: YAML, JSON, RDF/XML, tag\:value flat text, and xlss spreadsheets.
+Open-Source Software Security
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-`SWID Tags <https://nvd.nist.gov/products/swid>`__ is an international XML-based standard used by commercial software publishers and has been published as the standard ISO/IEC 19770-2. The specification defines four types of SWID tags: primary, patch, corpus, and supplemental to describe a software component.
+Open-source code is present in Cloud Infrastructure software from host Operating System to virtualisation layer 
+components, the most obvious being represented by Linux, KVM, QEMU, OpenStack, and Kubernetes. Workloads components 
+can also be composed of open source code. The proportion of open-source code to an application source code can vary. 
+It can be partial or total, visible or not. Open-source code can be upstream code coming directly from open-source 
+public repositories or code within a commercial application or network function. 
+
+The strength of open-source code is the availability of code source developed by a community which maintain and improve it. 
+Open-source code integration with application source code helps to develop and produce applications faster. But, in return, 
+it can introduce security risks if a risk management DevSecOps approach is not implemented. The GSMA white paper, “Open 
+Networking & the Security of Open Source Software Deployment - Future Networks”, alerts on these risks and addresses the 
+challenges coming with open-source code usage. Amongst these risks for security, we can mention a poor quality code 
+containing security flaws, an obsolete code with known vulnerabilities, and the lack of knowledge of open source 
+communities’ branches activity. An active branch will come with bugs fixes, it will not be the case with an inactive branch. 
+The GSMA white paper develops means to mitigate these security issues.
+
+Poor code quality is a factor of risk. Open-source code advantage is its transparency, code can be inspected by tools with 
+various capabilities such as open-source software discovery and static and dynamic code analysis.
+
+A dedicated internal isolated repository separated from the production environment must be used to store vetted open-source 
```

Comment:
We may need to explain "internal" here. Each member in the whole chain of software creation must follow this rule (I mean each provider of a component in a software product).

Reply:
@ulikleber You are right, this principle applies to each actor integrating an open-source component into his software product.
I change the sentence to "Each actor in the whole chain of software production must use a dedicated internal isolated repository separated from the production environment to store..."